itests/empty_test.go

```diff
@@ -0,0 +1,137 @@
+package itests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/filecoin-project/curio/deps"
+	"github.com/filecoin-project/curio/deps/config"
+	"github.com/filecoin-project/curio/harmony/harmonydb"
+	"github.com/filecoin-project/curio/tasks/seal"
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lotus/api"
+	miner2 "github.com/filecoin-project/lotus/chain/actors/builtin/miner"
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/xerrors"
+)
+
+func TestXYZ(t *testing.T) {
```

Comment:
this is the most opaque test: 
rename
add doc 
break into sections, funcs, setup helpers, etc.

Reply:
This is not a real PR. It was opened so @magik6k can review the network bootstrapping parts. Once network parts are resolved, I will rearrange everything.